packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.service.js

```diff
@@ -208,6 +208,35 @@ export default /* @ngInject */ function Ip(
         }),
       );
 
+  this.getIpsForIpV6 = (ipBlock, serviceName) =>
+    $q
+      .all({
+        bridgedSubrange: $http
+          .get(
+            `${swsProxypassPath}/vrack/${window.encodeURIComponent(
+              serviceName,
+            )}/ipv6/${window.encodeURIComponent(ipBlock)}/bridgedSubrange`,
```

Comment:
No need to call sws you can directly call ipv6 here "/vrack/..."

Reply:
ok (y)